dapper/tools/rounding.py

```diff
@@ -83,8 +84,10 @@ def __str__(self):
             # Rounding to fallback (rc.sigfig) already took place
             return f"{v} ±{c}"
         if c == 0:
-            # 0 (i.e. not 1e-300) never arises "naturally" => Treat it "magically"
-            # by truncating to a default. Also see https://stackoverflow.com/a/25899600
+            # 0 (i.e. not 1e-300) never arises "naturally"
+            # => Treat it "magically"
+            # by truncating to a default.
+            # Also see https://stackoverflow.com/a/25899600
```

Comment:
There is no need for these line breaks. The max_line_length is set to 88 is pyproject.toml

Reply:
Thanks, we can restore this.